src/analysis/complex/basic.lean

```diff
@@ -103,37 +103,38 @@ le_antisymm (linear_map.mk_continuous_norm_le _ zero_le_one _) $
 calc 1 = ∥im_clm I∥ : by simp
    ... ≤ ∥im_clm∥ : unit_le_op_norm _ _ (by simp)
 
-/-- The complex-conjugation function from `ℂ` to itself is an isometric linear map. -/
-def conj_li : ℂ →ₗᵢ[ℝ] ℂ := ⟨conj_lm, λ x, by simp⟩
+/-- The complex-conjugation function from `ℂ` to itself is an isometric linear equivalence. -/
+def conj_li : ℂ ≃ₗᵢ[ℝ] ℂ := ⟨conj_alg_equiv.to_linear_equiv, abs_conj⟩
 
-@[simp] lemma conj_li_apply (z : ℂ) : conj_li z = conj_lm z := rfl
-
-/-- Continuous linear map version of the conj function, from `ℂ` to `ℂ`. -/
-def conj_clm : ℂ →L[ℝ] ℂ := conj_li.to_continuous_linear_map
+@[simp] lemma conj_li_apply (z : ℂ) : conj_li z = conj z := rfl
 
 lemma isometry_conj : isometry (conj : ℂ → ℂ) := conj_li.isometry
 
-@[continuity] lemma continuous_conj : continuous conj := conj_clm.continuous
+@[continuity] lemma continuous_conj : continuous conj := conj_li.continuous
 
-@[simp] lemma conj_clm_coe : (coe (conj_clm) : ℂ →ₗ[ℝ] ℂ) = conj_lm := rfl
+/-- Continuous linear equiv version of the conj function, from `ℂ` to `ℂ`. -/
```

Comment:
```suggestion
/-- Continuous linear equiv version of the conj function, from `ℂ` to `ℂ`.

TODO: remove the `m` from this name as it refers to the previous type `continuous_linear_map`. -/
```
Unless you want to push through with the rename in this PR

Reply:
OK, let me do the renaming now it builds; there are a few others that need renaming too.